package.json

```diff
@@ -120,7 +120,7 @@
     "terra-status-view": "^4.10.0",
     "terra-theme-context": "^1.0.0",
     "terra-theme-provider": "^4.0.0",
-    "uuid": "^3.0.0",
+    "uuid": "^8.0.0",
```

Comment:
There were non-passive import changes from v3 -> v8. The import declaration will need to change in the files consuming uuid.

https://github.com/uuidjs/uuid#quickstart

```
import { v4 as uuidv4 } from 'uuid';
```

Reply:
Done